presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -462,12 +463,22 @@ int32_t SystemConfig::queryMemoryGb() const {
   return optionalProperty<int32_t>(kQueryMemoryGb).value();
 }
 
+int32_t SystemConfig::queryReservedMemoryGb() const {
```

Comment:
Why is this 32 bit vs 64 bit usage in other places ? Might be good to keep it uniform.

Reply:
It is just to be the same type as queryMemoryGb. We could uniform these config settings in followup. Thanks!